bootstrap/bootstrap.go

```diff
@@ -87,11 +117,11 @@ type bootstrap struct {
 	accessTokenDurationSeconds uint64
 	uriBasePath                string
 
-	cfg      *config.Config
-	managers *managers.Managers
+	Cfg      *config.Config
+	Managers *managers.Managers
 }
 
-func init() {
+func Boot(ctx context.Context, bsConf *Config, serverConf *config.Config) (*bootstrap, error) {
```

Comment:
This is missing a comment. Additionally this is returning a private type which is not that nice. I think the private type `bootstrap` should stay private and this should return a `Bootstrap` interface.